README.md

```diff
@@ -22,6 +22,7 @@ transpiled es6 classes rather than `createClass` as of v6.
 ## Sections covered in this README
 
 * [Installation](#installation)
+* [Usage](#usage)
```

Comment:
we might as well also get those nested entries in, as it'll let people laser-focus on what they're looking for. Probably indented?

Reply:
Good idea, usage headings are now indented under Usage)